src/components/LineChart/LineChart.props.ts

```diff
@@ -0,0 +1,11 @@
+export interface IChartProps {
+    title: string;
+    width: number;
+    height: number;
+    data: IChart[];
+}
+
+export interface IChart {
```

Comment:
Could we rename this interface? IChart does not really say that it is related to Series points. IChartSeriesPoint? or something like that perhaps